spec/mailers/user_mailer_spec.rb

```diff
@@ -11,6 +11,7 @@
     ActionMailer::Base.delivery_method = :test
     ActionMailer::Base.perform_deliveries = true
     ActionMailer::Base.deliveries = []
+    ActionMailer::Base.default_url_options[:host] = Rails.application.routes.default_url_options[:host]
```

Comment:
I notice that this var is set in `config/environments/test.rb`. I wonder if we should set it globally there?